web/main/models.py

```diff
@@ -1653,6 +1656,41 @@ def export_content(self, request):
             return rich_text_export(self.resource.content, request=request, id_prefix=str(self.id))
         return self.resource.content
 
+    def get_ali_license_text(self):
+        licensed_materials = [
+            {
+                "title": "Restatements of the Law",
+                "copyright_year": 2014,
+                "match_words": [
+                    "restatement",
+                    "restatements",
+                    "contracts",
```

Comment:
Matching "contracts" seems like it'll be too broad. @jcushman do you have a sense of this?